docs/features/software-templates/writing-custom-actions.md

```diff
@@ -189,7 +194,7 @@ export default async function createPlugin(
 }
 ```
 
-### Register Action With New Backend System
+### Register Action With New Backend System, Backstage Version >= 1.24.0
```

Comment:
Think that we should flip this around a little bit, and just call this `Register Actions` and then the section above becomes `Register Actions with the Legacy Backend System`?

Reply:
Makes sense to me! I'll make that change.